modules/rosa-create-objects.adoc

```diff
@@ -6,9 +6,55 @@
 [id="rosa-create-objects_{context}"]
 = Create objects
 
-
 This section describes the `create` commands for clusters and resources.
 
+[id="rosa-create-account-roles_{context}"]
```

Comment:
@pneedle-rh Side question - did you guys ever consider automatically generating the CLI docs, like how we do for `oc`? Not sure if it's possible, to get all the info you need, just wanted to ask!

Reply:
@bergerhoffer thank you for your detailed review and for this suggestion. I think it is a great idea. Any details about how we could implement this would be very much appreciated, thanks!

@AndrewJones-RH and @bmcelvee is this something that we are looking in to?

Reply:
@bergerhoffer based on conversations within the team today, I don't think we have looked into this yet, but that we we would like to.

Reply:
@pneedle-rh I put together a doc on how we auto-generate the `oc` CLI docs, which includes references to the code in the `oc` repo that makes it possible. I'll share that link with you on slack!